types/src/system/auction/error.rs

```diff
@@ -98,6 +98,7 @@ pub enum Error {
     MissingDelegations = 22,
     /// The validators returned by the consensus component should match
     /// current era validators when distributing rewards.
+    /// **Deprecated**
```

Comment:
```suggestion
    #[deprecated(since="1.4.0")]
```

Reply:
@xcthulhu: Looks like I can't do that because of the options we run Clippy with:

```
note: `-D deprecated` implied by `-D warnings`
```

I'd rather not change the CI settings for this?

Reply:
We'd have to annotate `#[allow(deprecated)]` where we use this; this would also force us to do an audit to make sure we aren't accidentally using it or testing for it anywhere.

Reply:
The problem I see with this is that the `deprecated` would work with the code/software versioning while this particular deprecation is about protocol versioning. The two are currently equivalent but they won't be in the future.

Reply:
I actually tried `#[allow(deprecated)]`, too, and it didn't work: https://github.com/casper-network/casper-node/pull/1950/commits/c1d09899fd6f8a74d858b27d7b2c2c4ab246cb09
Probably because of the explicit `-D warnings` argument?